src/routes/profile/[name]/+page.ts

```diff
@@ -0,0 +1,8 @@
+export function load({ params }) {
+	return {
+		props: {
+			name: params.name,
+			email: 'test-email@domain.com'
+		}
+	};
+}
```

Comment:
Hier nur temporär, in der `+page.ts` wird die Load-Funktion deklariert, durch die benötigte Daten für die Seite vorgeladen werden können. An der Stelle würden wir später also unser Backend nach den Userdaten anpingen